DQMOffline/Trigger/python/ExoticaMonitoring_Client_cff.py

```diff
@@ -9,13 +12,35 @@
         "effic_met          'MET turnON;            PF MET [GeV]; efficiency'     met_numerator          met_denominator",
         "effic_met_variable 'MET turnON;            PF MET [GeV]; efficiency'     met_variable_numerator met_variable_denominator",
         "effic_metPhi       'MET efficiency vs phi; PF MET phi [rad]; efficiency' metPhi_numerator       metPhi_denominator",
+        "effic_ht          'HT turnON;            PF HT [GeV]; efficiency'     ht_numerator          ht_denominator",
```

Comment:
@mtosi Could you clarify? Maybe @sudhaahuja could comment on any questions you have about this.